ossdl-cdn.php

```diff
@@ -272,13 +276,47 @@ function scossdl_off_options() {
 	$example_cdn_uri .= '/wp-includes/js/jquery/jquery-migrate.js';
 	$example_cdn_uri  = esc_url( $example_cdn_uri );
 	?>
+		<h3><?php _e( 'Jetpack CDN' ); ?></h3>
+		<p><?php printf(
+			__( 'The free %1$sJetpack plugin%2$s has a %3$sSite Accelerator%2$s feature that is easier to use than the CDN functionality in this plugin. However files will be cached "forever" and will not update if you update the local file. Files will need to be renamed to refresh them. The %3$sJetpack documentation%2$s explains more about this.', 'wp-super-cache' ),
+			'<a href="https://jetpack.com/">',
+			'</a>',
+			'<a href="https://jetpack.com/support/site-accelerator/">'
+		); ?></p>
+	<?php
+	if ( class_exists( 'Jetpack' ) ) {
+		if ( Jetpack::is_module_active( 'photon' ) ) {
+			?><p><strong><?php printf(
+				__( 'You already have Jetpack installed and %1$sSite Accelerator%2$s enabled on this blog. The CDN here is disabled to avoid conflicts with Jetpack.', 'wp-super-cache' ),
+				'<a href="https://jetpack.com/support/site-accelerator/">',
+				'</a>'
+			); ?></strong></p><?php
+		} else {
+			?><p><?php printf(
+				__( 'You already have Jetpack installed but %1$sSite Accelerator%2$s is disabled on this blog. Enable it on the %3$sJetpack settings page%2$s.', 'wp-super-cache' ),
+				'<a href="https://jetpack.com/support/site-accelerator/">',
+				'</a>',
+				'<a href="' . admin_url( 'admin.php?page=jetpack#/settings' ) . '">'
+			); ?></p><?php
+		}
+	} else {
+			?><p><?php printf(
+				__( '%1$sJetpack%2$s is not installed on your site. The Site Accelerator feature is free to use on any WordPress site and offers the same benefit as other CDN services. You should give it a try!', 'wp-super-cache' ),
```

Comment:
This one might be better as something like "Jetpack's Site Accelerator feature is free to use...".

Maybe we can also just link internally/directly to the plugin install page rather than sending them off to jetpack.com? Although I guess the only way to do that is actually via a search, which is not great: `wp-admin/plugin-install.php?s=jetpack&tab=search&type=term`

Reply:
plugin-install.php will redirect to wp-admin/network/plugin-install.php on an MS install, but Jetpack is one of the featured plugins so that isn't too bad.

Reply:
I updated the code with a link to plugin-install.php. I didn't change the second sentence though because Jetpack is mentioned so much in the previous text.